pombola/core/views.py

```diff
@@ -611,6 +611,31 @@ def handle_approx_date(date):
 
         return response
 
+    elif request.GET.get('format') == 'popolo':
+
+        persons = []
+
+        for position in positions:
+
+            person = position.person
+
+            # Only append the person if they have an email address
+            email_query = person.contacts.filter(kind__slug='email').order_by('-preferred')
+            if email_query.exists():
+                persons.append(
+                    {
+                        'id': request.build_absolute_uri(person.get_absolute_url())
+                                    .replace('http://', 'https://'),
+                        'name': person.name,
+                        'email': email_query.first().value,
+                        'contact_details': []
+                    }
+                )
+
+        response = JsonResponse({'persons': persons})
```

Comment:
Looks like this is currently returning duplicates where people hold more than one position. I think that's because a person can hold multiple positions, so the same person appears more than once in the list of positions.